examples/chart/teleport/values.yaml

```
@@ -150,10 +150,16 @@ config:
       # Used if  proxy.tls.usetlssecret is set to true, otherwise the values are not included in teleport.yaml
       https_key_file: /var/lib/certs/tls.key
       https_cert_file: /var/lib/certs/tls.crt
+      # Specify different address for the ssh and tunnel public address if different then the config.public_address
+    # ssh_public_addr: teleportssh.example.com
+    # tunnel_public_addr: teleporttunnel.example.com
```

```suggestion
      # ssh_public_addr: teleportssh.example.com
      # tunnel_public_addr: teleporttunnel.example.com
```

Sorry, looks like I messed the spacing of this up before.